src/tex/appendix.tex

```diff
@@ -588,6 +587,24 @@ \subsection{Human baseline} \label{sec:human_baseline}
     \label{fig:tool_use}
 \end{figure}
 
+\clearpage
+\subsection{Tool augmented models} \label{sec:react-environment}
+In addition to directly prompting LLMs, we also investigated the performance of tool-augmented systems.
+For this, we investigated the two models which showed the best overall results, \GPTFourO and \ClaudeThreeFiveSonnet (\oone overperformed both models but is not recommended using this model with such \enquote{reasoning} prompts such as ReAct\autocite{yao2022react} or CoT\autocite{wei2023cot}). 
+For both models, we created a ReAct-style tool augmentation environment in which models had access to WolframAlpha, the ArXiv API, the Wikipedia access, and a web search (using Brave browser API).
+We based the election of these tools based in the most used tools by the humans (see \Cref{fig:tool_use}).
+Additionally, we added two specific tools to convert IUPAC names to SMILES, and SMILES to IUPAC names.
+These conversion tools allow us to better understand how the agents perform for some specific questions in this agent environment configurations.
+We implemented the systems using Langchain with the default ReAct prompt and constrained the system to a maximum of ten LLM calls.
+
+While for \ClaudeThreeFiveSonnet the overall performance increased (less than 1\%), they show a sloped decrease for \GPTFourO compared with the \llm without tools (\Cref{tab:performance_table}). 
```

Comment:
```suggestion
While for \ClaudeThreeFiveSonnet the overall performance increased (by less than 1\%), we observe a decrease in performance for \GPTFourO compared with the \llm without tools (\Cref{tab:performance_table}). 
```

Reply:
@MrtinoRG as far as I see there is no improvement for Claude? it is almost the same number, without the agent being slightly better?